test/functional/interface_zmq_dash.py

```
@@ -218,8 +221,8 @@ def validate_recovered_sig(request_id, msg_hash):
         # Sign an arbitrary and make sure this leads to valid recovered sig ZMQ messages
         sign_id = uint256_to_string(random.getrandbits(256))
         sign_msg_hash = uint256_to_string(random.getrandbits(256))
-        for mn in self.get_quorum_masternodes(self.quorum_hash):
-            mn.node.quorum("sign", self.quorum_type, sign_id, sign_msg_hash)
+        for mn in self.get_quorum_masternodes(self.quorum_hash): # type: MasternodeInfo
```

Pylint (3.3.7)

[error] Instance of 'DashZMQTest' has no 'quorum_hash' member

(E1101)